us-geological-survey.csl

```diff
@@ -121,8 +150,8 @@
               <et-al term="and others"/>
               <label form="short" prefix=" "/>
             </names>
-            <text variable="container-title" prefix=" " suffix=","/>
-            <text variable="collection-title" prefix=" " suffix=","/>
+            <text variable="event" prefix=" " suffix=" --"/>
```

Comment:
@jbellino-usgs, you can update this pull request yourself by clicking the pencil icon at https://github.com/jbellino-usgs/styles/blob/patch-1/us-geological-survey.csl. If you need an en-dash, one way is to just copy and paste it from the internet (e.g. from http://en.wikipedia.org/wiki/Dash, or directly from here: "–").